lib/cfg-parser.c

```diff
@@ -452,10 +447,52 @@ cfg_process_flag(CfgFlagHandler *handlers, gpointer base, const gchar *flag_)
   return FALSE;
 }
 
+gboolean
+cfg_validate_yesno(const gchar *yesno, gboolean *value)
```

Comment:
Have you considered merging the `cfg_validate_yesno` and `cfg_process_yesno` ?

Reply:
Yep, I just did that :P